gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/jupiter/core/v4/endpoint/DefaultEndpointManager.java

```diff
@@ -48,7 +47,9 @@ public class DefaultEndpointManager extends AbstractService<EndpointManager> imp
     private ManagedEndpointGroup defaultGroup;
     private final Map<String, ManagedEndpointGroup> groupsByName;
     private final Map<String, ManagedEndpoint> endpointsByName;
+    private final Set<ManagedEndpoint> disabledEndpoints;
     private final Map<String, String> endpointVariables;
+    private final Map<String, BiConsumer<Event, ManagedEndpoint>> listeners; // TODO [ELE] where is it used ?
```

Comment:
TODO is outdated, it was to reminde me to ask question to archi team.
Basically, this listeners map is updated in the HealthCheckApiService but the listeners are never executed. This is because they have been introduced in prevision of the service discovery that may have to add/remove endpoint on which we will have to start/Stop the HC.